book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+    rental.book = self
+  end
```

Comment:
- [ ] @aradradev Great work in your association, however, to achieve `the 'has-many' association` between books and person through rentals, we need to pass the `person, date and book` itself as parameters to create the new rental.

Reply:
- [ ] @aradradev I didn't see your implementation of the association in your person class, kindly consider adding the association in order to achieve the  the 'has-many' association between books and person through rentals